zipkin-receiver-kafka/src/test/scala/com/twitter/zipkin/receiver/kafka/KafkaProcessorSpec.scala

```diff
@@ -1,47 +1,31 @@
 package com.twitter.zipkin.receiver.kafka
 
-import com.twitter.zipkin.receiver.test.kafka.{TestUtils, EmbeddedZookeeper}
+import com.github.charithe.kafka.KafkaJunitRule
+import com.twitter.util.{Await, Future, Promise}
 import com.twitter.zipkin.common._
+import com.twitter.zipkin.conversions.thrift.thriftSpanToSpan
 import com.twitter.zipkin.thriftscala.{Span => ThriftSpan}
-import com.twitter.zipkin.conversions.thrift.{thriftSpanToSpan, spanToThriftSpan}
-import com.twitter.util.{Await, Future, Promise}
-import com.twitter.scrooge.BinaryThriftStructSerializer
-
-import org.junit.runner.RunWith
-import org.scalatest.FunSuite
-import org.scalatest.BeforeAndAfter
-import org.scalatest.junit.JUnitRunner
-
-import kafka.consumer.{Consumer, ConsumerConnector, ConsumerConfig}
-import kafka.message.Message
 import kafka.producer._
-import kafka.serializer.Decoder
-import kafka.server.KafkaServer
+import org.junit.{ClassRule, Test}
+import org.scalatest.junit.JUnitSuite
 
-import com.twitter.zipkin.collector.{SpanReceiver, ZipkinQueuedCollectorFactory}
-import java.io._
+object KafkaProcessorSpecSimple {
+  // Singleton as the test needs to read the actual port in use
+  val kafkaRule = new KafkaJunitRule()
```

Comment:
Pretty cool, [kafka-unit](https://github.com/chbatey/kafka-unit) allows you to delete all that test code